src/main/webapp/app/shared/image/secured-image.component.ts

```diff
@@ -44,6 +44,7 @@ export class SecuredImageComponent implements OnChanges {
     @Input() mobileDragAndDrop: boolean;
     @Input() private src: string;
     @Input() cachingStrategy = CachingStrategy.SESSION_STORAGE;
+    @Input() alt = '';
```

Comment:
This input was missing, so the alt tag was not displayed (not a major issue). However this is a problem of the schema setting as the compiler did not detect the issue.